fetch/cosmwasm_contract/src/contract.rs

```diff
@@ -3,12 +3,22 @@ use cosmwasm_std::{
     HandleResponse, HandleResult, HumanAddr, InitResponse, Querier, StdError, StdResult, Storage,
 };
 
-use crate::access_control::{
-    ac_add_role, ac_get_owner, ac_have_role, ac_revoke_role, ac_set_owner, AccessRole,
+use crate::access_control::{ac_add_role, ac_have_role, ac_revoke_role, ac_set_owner, AccessRole};
+use crate::error::{
+    ERR_ACCESS_CONTROL_DONT_HAVE_ROLE, ERR_ACCESS_CONTROL_ONLY_ADMIN,
+    ERR_ACCESS_CONTROL_ONLY_RELAYER, ERR_ALREADY_REFUNDED, ERR_CAP_EXCEEDED, ERR_CONTRACT_PAUSED,
+    ERR_EON, ERR_INVALID_SWAP_ID, ERR_RA_ALLOWANCE_EXCEEDED, ERR_SUPPLY_EXCEEDED,
+    ERR_SWAP_LIMITS_INCONSISTENT, ERR_SWAP_LIMITS_VIOLATED, ERR_UNRECOGNIZED_DENOM,
+};
+use crate::msg::{
+    CapResponse, DenomResponse, HandleMsg, InitMsg, PausedSinceBlockResponse, QueryMsg,
+    RelayEonResponse, ReverseAggregatedAllowanceResponse, RoleResponse, SupplyResponse,
+    SwapMaxResponse, Uint128,
 };
-use crate::msg::{HandleMsg, InitMsg, QueryMsg, RoleResponse, Uint128, RelayEonResponse, ReverseAggregatedAllowanceResponse, SupplyResponse};
 use crate::state::{config, config_read, refunds_add, refunds_have, State};
 
+pub const DENOM: &str = "atestfet";
```

Comment:
This should rather by named `DEFAULT_DENOM` to make it clear its purpose 
And value should rather be "afet".